packages/injected/src/icons/tokenpocket.ts

```diff
@@ -1,17 +1,25 @@
 export default `<?xml version="1.0" encoding="utf-8"?>
-<svg width="100%" height="100%" version="1.1" id="图层_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
-	 viewBox="0 0 1024 1024" style="enable-background:new 0 0 1024 1024;" xml:space="preserve">
-<style type="text/css">
-	.st0{fill:#2980FE;}
-	.st1{opacity:0.6;fill:#FFFFFF;}
-	.st2{fill:#FFFFFF;}
-</style>
-<rect x="0.5" y="-0.5" class="st0" width="1024" height="1024"/>
-<g>
-	<path class="st1" d="M464.1,389.7h-19.1V253.3h-252c-4.8,0-8.7,3.9-8.7,8.7v192.3l0,0l0,0c0,4.8,3.9,8.7,8.7,8.7h96.2v349.7l0,0
-		l0,0c0,4.8,3.9,8.7,8.7,8.7h0l0,0h157.4c4.8,0,8.7-3.9,8.7-8.7l0,0l0,0V389.7z"/>
-	<path class="st2" d="M630.8,201.5h-60.5l0,0H404.2l0,0c-4.8,0-8.7,3.9-8.7,8.7V761l0,0l0,0c0,4.8,3.9,8.7,8.7,8.7l0,0h157.4
-		c0,0,0,0,0,0c4.8,0,8.7-3.9,8.7-8.7V621.1h60.5c115.9,0,209.8-93.9,209.8-209.8C840.6,295.4,746.7,201.5,630.8,201.5z"/>
-</g>
+<svg width="120" height="120" viewBox="0 0 1024 1024" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
```suggestion
<svg width="100%" height="100%" viewBox="0 0 1024 1024" fill="none" xmlns="http://www.w3.org/2000/svg">
```
@chendatony31 Recommend setting the H/W to 100% for future flexibility - not required 😄 

<img width="797" alt="Screen Shot 2022-05-10 at 10 13 54 AM" src="https://user-images.githubusercontent.com/24457880/167674768-100b4f27-d6d2-435f-ad21-b10520ce793f.png">